### PR TITLE
Add key-values format to data task

### PR DESCRIPTION
**Pull request by arjan0307**

### CODE REVIEW

File: lib/i18n/tasks/command/options/data.rb [hunk 1]
```diff
@@ -85,6 +85,8 @@ def print_forest(forest, opts, version = :show_tree)
             puts forest.inspect
           when 'keys'
             puts forest.key_names(root: true)
+          when 'key-values'
+            puts forest.key_values(root: true).map { |kv| kv.join(' ') }
```
Comment (glebm) [inline lib/i18n/tasks/command/options/data.rb:89]: The key can contain spaces so perhaps it's better join with a `\t`?

Comment (arjan0307) [reply lib/i18n/tasks/command/options/data.rb:89]: Good point, although wouldn't it make more sence to use a colon?

Comment (glebm) [reply lib/i18n/tasks/command/options/data.rb:89]: A colon can also be part of the key name

Comment (arjan0307) [reply lib/i18n/tasks/command/options/data.rb:89]: Okay, I'll make it a `\t`. Couldn't that also be a part of the key name though? As far as I can tell every character is allowed as long as it's properly escaped.

Comment (glebm) [reply lib/i18n/tasks/command/options/data.rb:89]: True but `\t` is unlikely whereas I've seen `:` and ` ` in the bug reports here

**Pull request merged**